File: Save Raj from Bajrang Dal .cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    long long int n,i,sum=0;
    cin>>n;
    long long int a[n];
    for(i=0;i<n;i++)
    {
        cin>>a[i];
    }
    sort(a,a+n);
    {
        for(i=0;i<n/2;i++)
        {
            int s=a[i]+a[n-1-i];
            sum+=(s*s);
        }
    }
    cout<<sum;
}

File: Solutions/Java/Save Raj from Bajrang Dal.java
import java.util.*;
import java.io.*;
import java.math.*;
public class Waw{
    
    public static void main(String[] args) throws Exception{
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        
        long[] a = new long[n];
        for(int i=0;i<n;i++) a[i] = sc.nextLong();
        Arrays.sort(a);
        long x,y,sum=0;
        for(int i=0;i<n/2;i++){
            x = a[i];
            y = a[n-i-1];
            sum += (x+y)*(x+y);
        }
        System.out.println(sum);
    }
    

    
    
}